src/rpm/headers/types.rs

```diff
@@ -277,6 +280,16 @@ impl FileOptionsBuilder {
         Ok(self)
     }
 
+    /// Direct which aspects of the file you would like RPM to verify.
+    ///
+    /// By default, every aspect of the file will be checked.
+    ///
+    /// See: `%verify` from specfile syntax
+    pub fn verify(mut self, flags: FileVerifyFlags) -> Self {
```

Comment:
I can see how it would sound like an active operation in isolation, but since it's on the `FileOptions` type I think the meaning is probably sufficiently clear